typescript/nuxtjs-prisma-nuxt-module/lib/prisma.ts

```diff
@@ -0,0 +1,29 @@
+import { Prisma, PrismaClient } from '@prisma/client'
+
+const prismaClientSingleton = () => {
+  // Custom extended Prisma ORM instance with client extensions, adding a method to verify the existence of a database entry.
+  return new PrismaClient().$extends({
```

Comment:
do we need a client extension in this example?

Reply:
@jharrell I added the client extension example to show the custom `prisma.ts` lib exists. I can elaborate further on why.

Reply:
Elaborating 👍 !

Reply:
@jharrell elaborated in the README.md